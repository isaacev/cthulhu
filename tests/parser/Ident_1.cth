#[entry]
fn main() -> () {
  let fn = "abcdef";
}
