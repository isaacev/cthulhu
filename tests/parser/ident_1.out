
  USE OF RESERVED WORD  ------------------------------------------------------

    : ident_1
  3 |   let fn = "abcdef";
    |       ^^ reserved words cannot be used as identifiers

